test/types/index.ts

```diff
@@ -127,6 +129,12 @@ server.addHook('onResponse', function (req, reply, next) {
   }, 100)
 })
 
+server.addHook('preSerialization', function (req, reply, payload, next) {
+  this.log.debug('`this` is not `any`')
+  console.log(`${req.req.method} ${req.req.url}`)
+  next(undefined, payload)
+})
+
```

Comment:
I think TS is missing a test for the new option to  `route`.

Reply:
You're right. I added it now.